src/libbacktrace/pecoff.c

```diff
@@ -602,7 +602,7 @@ coff_add (struct backtrace_state *state, int descriptor,
   const b_coff_section_header *sects;
   struct backtrace_view str_view;
   int str_view_valid;
-  uint32_t str_size;
+  size_t str_size;
```

Comment:
Wait, wait, wait! This `uint32_t` is our fix for an uninitialized variable use, see https://github.com/rust-lang/rust/pull/30908#issuecomment-172008039
Rust-specific patch to libbacktrace looks like this: https://github.com/rust-lang/rust/pull/30908/commits/55e2b7e1b4606ae0bc684293f011b7006d1f1258